tests/v6/utils-e2e-v6.ts

```diff
@@ -1,18 +1,12 @@
-import { Interface } from '@ethersproject/abi';
-import { Address, OptimalRate, SwapSide } from '@paraswap/core';
-import Erc20ABI from '../../src/abi/erc20.json';
-import { generateConfig } from '../../src/config';
-import { Network, MAX_UINT, ETHER_ADDRESS } from '../../src/constants';
-import {
-  TenderlySimulation,
-  TransactionSimulator,
-} from '../tenderly-simulation';
+/* eslint-disable no-console */
```

Comment:
we could instead turn off this rule for all the tests by adding this to the lint config
```  
{
  overrides: [
    {
      files: ['tests/**/*.ts', '*.test.ts'],
      rules: {
        'no-console': 'off',
      },
    },
  ],
}
  ```